Malayalam/sasi/pling.md

```diff
@@ -1 +1,3 @@
-പ്ലിംഗ്  എന്നാൽ ചമ്മുക , അല്ലാതെ  എന്ത് . ചുരുക്കി പറഞ്ഞാൽ ശശി എന്ന് നാമമുള്ളവൻ എന്നും പ്ലിങ്ങും 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.